include/robot_interfaces/sensors/pylon_driver.hpp

```diff
@@ -88,11 +127,16 @@ class PylonDriver : public SensorDriver<CameraObservation>
                                             CV_8UC3,
                                             (uint8_t*)pylon_image_.GetBuffer());
             }
+            else
+            {
+                throw "Failed to access images from the camera.";
```

Comment:
Better throw an exception than just a string
```suggestion
                throw std::runtime_error("Failed to access images from the camera.");
```